packages/server-wallet/src/__test__/create-and-close-channel/fake-funding.test.ts

```diff
@@ -86,7 +86,9 @@ it('Create a fake-funded channel between two wallets ', async () => {
   // after joinChannel, B double-signs PreFund0
   const bJoinChannelOutput = await b.approveObjective(
     // eslint-disable-next-line
-    bProposeChannelPushOutput.objectivesToApprove![0].objectiveId
+    bProposeChannelPushOutput.objectivesToApprove![0].objectiveId,
+    // eslint-disable-next-line
+    bProposeChannelPushOutput.objectivesToApprove![0].type
```

Comment:
Hm we may want to do some assertion on `objectivesToApprove` to avoid using `!`

Reply:
We could have a `getOpenChannelObjectiveFor(targetChannelId:string, singleChannelOutput: SingleChannelOutput)` test utility that throws if there isn't exactly 1 objective of the correct type. We then assert on the output of this function.